server/home/home-common/src/main/java/io/holoinsight/server/home/common/util/retrofit/RetrofitProxy.java

```diff
@@ -68,15 +68,15 @@ protected void printBefore(Method method, Object[] args) {
     } else {
       invokeInfo.append("Invoke with 0 params.");
     }
-    // logger.warn(invokeInfo.toString());
+    log.warn(invokeInfo.toString());
   }
 
   protected void printAfter(long start, String url, Method method, Response response) {
 
     String invokeInfo = "Invoke INFO: method=[" + method.getName() + "], url=[" + url + "], cost=["
         + (System.currentTimeMillis() - start) + "], " + "result=[" + response.isSuccessful()
         + "], code=[" + response.code() + "]";
-    // logger.warn(invokeInfo);
+    log.warn(invokeInfo);
```

Comment:
use log placeholder, for example: log.warn(" ... {} ... {}")

Reply:
> 
have completed